content/micropython/06.communication/spi/spi.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Serial Peripheral Interface (SPI)
+description: Learn how to use SPI with Micropython
+author: Pedro Sousa Lima
+---
+
+Serial Peripheral Interface, or **SPI**, is a widely used communication protocol for connecting devices. It enables high-speed, full-duplex communication over four primary wires, making it an excellent choice for applications requiring fast data exchange between microcontrollers and peripherals.
+
+## How SPI Works
+
+SPI communication relies on:
+
+- **MOSI** (Controller Out, Peripheral In): Sends data from the controller to the peripheral.
+
+- **MISO** (Peripheral Out, Controller In): Receives data from the peripheral to the controller.
+
+- **SCLK** (Serial Clock): Synchronizes data transmission.
+
+- **CS** (Chip Select): Selects which peripheral device to communicate with.
+
+![SPI Wiring](assets/spi.png)
+
+Unlike UART, SPI is synchronous, meaning it uses a clock signal for precise data timing. This enables faster and more reliable data exchange. SPI operates in a master-peripheral architecture, where a single controller device can communicate with one or multiple peripheral devices. Each peripheral is selected using the Chip Select (CS) line, ensuring that only the intended device responds to communication.
+
+## Key Features of SPI
+
+-**High-Speed Communication:** Supports much higher data rates than UART, making it suitable for applications requiring rapid data transfer.
```

Comment:
The lists are not being rendered correctly, you need to add a space between the line (-) and the item (*). The first list is correct, check the other two